src/shared/components/case-editor/case-editor.module.ts

```diff
@@ -53,6 +55,7 @@ import { ErrorsModule } from '../error/errors.module';
         CaseEditFormComponent,
         CaseEditSubmitComponent,
         CaseCreateComponent,
+        CreateCaseFiltersComponent,
```

Comment:
I do not think we need this import here. It won't be used in this module.

Reply:
We should have a new CreateCaseFiltersModule with all necessary providers declaring one CreateCaseFilters component and exporting.

Reply:
I will do as suggested and do the "ccd-case-management-web" PR as part of this change. Therefore, will need a little more time.

Reply:
Both PR's are now in place.